.buildkite/release-builds.yml

```diff
@@ -11,10 +11,18 @@ steps:
     command: ".buildkite/commands/release-build-wordpress.sh $BETA_RELEASE"
     plugins: [$CI_TOOLKIT_PLUGIN]
     notify:
-    - slack: "#build-and-ship"
+      - slack: "#build-and-ship"
```

Comment:
On seeing this I wondered "How has it worked all this time?!" but it turns out YAML (or at least the Ruby parser and Buildkite's parser) are flexible on array indentation:

![image](https://github.com/user-attachments/assets/f86d96c9-9a22-4ef1-896e-397089215220)